From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Matti=20Lehtim=C3=A4ki?= <matti.lehtimaki@jolla.com>
Date: Sat, 9 Dec 2023 15:13:53 +0200
Subject: [PATCH] mkbootimg: Fix variable scope

---
 mkbootimg.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mkbootimg.c b/mkbootimg.c
index f72a2650..a19c083b 100644
--- a/mkbootimg.c
+++ b/mkbootimg.c
@@ -28,6 +28,7 @@
 
 static void *load_file(const char *fn, unsigned *_sz)
 {
+    char *data = 0;
     int fd = open(fn, O_RDONLY);
     if(fd < 0) return 0;
 
@@ -35,7 +36,6 @@ static void *load_file(const char *fn, unsigned *_sz)
     if(sz < 0) goto oops;
     if(lseek(fd, 0, SEEK_SET) != 0) goto oops;
 
-    char *data = 0;
     data = (char *)malloc(sz);
     if(data == 0) goto oops;
 
